packages/k/kitty/package.yml

```diff
@@ -1,9 +1,10 @@
 name       : kitty
-version    : 0.35.2
-release    : 70
+version    : 0.36.0
+release    : 71
 source     :
-    - https://github.com/kovidgoyal/kitty/releases/download/v0.35.2/kitty-0.35.2.tar.xz : b48ac902643bc225ec6ed830f496253ce4522dbe1d4b44fedc3106314c4fade2
+    - https://github.com/kovidgoyal/kitty/releases/download/v0.36.0/kitty-0.36.0.tar.xz : 41f29f42e260975a87433b0b72fc74360cfb77b153b63d35cabc0420fdbca550
     - git|https://github.com/simd-everywhere/simde.git : v0.7.6
+    - https://github.com/ryanoasis/nerd-fonts/releases/download/v3.2.1/NerdFontsSymbolsOnly.tar.xz : b7a3eee24d2b0910f3a07705f1e053221d2535f07c5a13614c4dead2620e9978
 license    : GPL-3.0-only
```

Comment:
The font being bundled is licensed under MIT, so let's add that to the license list:

```yaml
license    :
    - GPL-3.0-only
    - MIT
```